100=Driver.__new__()
101=Driver <{}> initialization completed
102=ERROR initializing driver <{}> following exception: {}

200=OptionsHandler.callHandlerMethod()
201=An unexpected exception: {} has occurred, producing a series of exceptions: {}

300=OptionsManager.__init__()
301=OptionManger has been correctly initialized.

310=OptionsManager.loadSetting()
311=OptionManger has been correctly reloaded.

320=OptionsManager.saveSetting()
321=OptionManger saved options {} and logging {}.


1000=Connect URL ERROR
1001=Authentication ERROR
1002=Authorization ERROR
1003=Setup ERROR
1004=Network ERROR
1005=DataBase ERROR
1006=Provider ERROR


1100=Driver.__init__()
1101=Driver initialization completed

1110=Driver.connect()
1111=Loading the driver for the URL {}
1112=Invalid protocol <{}>
1113=Invalid user account and/or password.
1114=Invalid user account <{}>
1115=Hsqldb version {} database is loaded, the user {} is connected.
1116=An error <{}> occurred while connecting with trace <{}>.


1200=DataSource.__init__()
1201=The data source is loading...
1202=Data source successfully loaded.

1250=DataBase.__init__()
1251=Database does not exist, creating new database.
1252=Database creation was successful.


1400=DataSource.getConnection()
1401=The user {} has abandoned the OAuth2 authorization wizard, the {} extension cannot continue without this authorization!!!


1500=User.__init__()
1501=User <{}> is loading...
1502=Searching for user <{}> in the database.
1503=User <{}> exists in the database. Trying to connect to the provider.
1504=User <{}> does not exist in the database. Trying to connect to the provider.
1505=User <{}> successfully loaded.
1506=The user {} has abandoned the OAuth2 authorization wizard, the {} extension cannot continue without this authorization!!!
1507=Can't connect to server <{}>. Is the network offline?
1508=Can't insert user {} in DataBase


1600=Provider.raiseForStatus()
1601=HTTP request <{}> returned status code {} for user {} on URL {} with content {}

1610=Provider.initUserBooks()
1611=Can't retrieve AddressBook for user {}, the server {} don't support AddressBook!!!

1620=Provider._getDiscoveryUrl()
1621=HTTP request <{}> for user {} on URL {} cannot retrieve <Location> on headers {}
1622=Despite {} attempts, impossible to find a discovery url other than: {}

1640=Provider._getNewUserId()
1641=User {} with password {} cannot connect to server {} at URL {}
1642=User {} cannot find on URL {} a CardDAV compatible server

1650=Provider._supportAddressbook()
1651=HTTP request <{}> for user {} on URL {} cannot retrieve <DAV> on headers {}

1660=Provider._getAddressbooksUrl()
1661=HTTP request <{}> for user {} on URL {} cannot parse content {}


1700=helper.checkConfiguration()
1701=The extension {} was not found. You must install this extension to use {}.
1702=Version {} of extension {} is obsolete. You need to update this extension to version: {}.
1703={} does not have the necessary API level to create the {} database. This level should be: <com.sun.star.sdb>

